### PR TITLE
Added support for intel.log zeek module

### PR DESCRIPTION
**Pull request by garanews**
Enrich the https://github.com/elastic/beats/pull/14150 supporting intel.log

Co-Authored-By: Arcuri Davide <dadokkio@gmail.com>

### CODE REVIEW

File: x-pack/filebeat/module/zeek/intel/_meta/fields.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+- name: intel
+  type: group
```
Comment (andrewkroh) [inline x-pack/filebeat/module/zeek/intel/_meta/fields.yml]: Please add `default_field: false` to this group.

**Comment by andrewkroh** [comment]
jenkins, test this

**Comment by andrewkroh** [comment]
jenkins, test this

File: x-pack/filebeat/module/zeek/intel/_meta/fields.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          type: keyword
+          description: >
+            The name of the node where the match was discovered.
+
+        - name: uid
+          type: keyword
+          description: >
+            If the data was discovered within a connection, the connection uid should go here to give context to the data. If the conn field is provided, this will be automatically filled out.
+
+        - name: f
```
Comment (andrewkroh) [inline x-pack/filebeat/module/zeek/intel/_meta/fields.yml]: I changed this to `f` from `fa_file` based on https://docs.zeek.org/en/stable/scripts/base/frameworks/intel/main.zeek.html#type-Intel::Seen. I don't have a log sample to verify it though.

**Comment by andrewkroh** [comment]
jenkins, test this

Will wait for CI to test these changes again then I'll merge it. Thanks!

File: x-pack/filebeat/module/zeek/intel/_meta/fields.yml (hunk 2 above)
Comment (garanews) [reply x-pack/filebeat/module/zeek/intel/_meta/fields.yml]:
```
{"ts":1574161961.329821,"uid":"Cm8HUc24IncCd8yW4c","id.orig_h":"172.16.160.22","id.orig_p":54919,"id.resp_h":"172.16.160.23","id.resp_p":445,"seen.indicator":"9052d06c6ac53471f8496263f8fef2eb","seen.indicator_type":"Intel::FILE_HASH","seen.where":"Files::IN_HASH","seen.node":"worker-2-4","matched":["Intel::FILE_HASH"],"sources":["Safe 32bit putty.exe"],"fuid":"FTsQFK277gbWqdNURh","file_mime_type":"application/x-dosexec","file_desc":""}

```

this is a row of mine, but  f ( f_a_file)  is not reported (also if I have https://docs.zeek.org/en/stable/scripts/base/frameworks/intel/files.zeek.html enabled)

**Pull request merged**